tests/simple_arg.c

```diff
@@ -0,0 +1,21 @@
+#include <assert.h>
+#include <stdlib.h>
+
+int
+check_fn(int one, int two)
+{
+    assert(one + two == 42);
+    return 0;
+}
+
+int
+main(int argc, char** argv)
+{
+	size_t sum = 0;
```

Comment:
Switch to spaces for indentation.

Reply:
Done in 17687ccaa1cff521ce4a1bdda26eee47e606b366.